src/RestClient/endpoints/V3/Themes/ThemeCustomTemplates.ts

```diff
@@ -0,0 +1,24 @@
+import { AxiosInstance } from 'axios';
+
+import { themesPath } from './Themes';
+import type { ThemeCustomTemplate } from './types';
+
+class ThemeCustomTemplates {
+  private client: AxiosInstance;
+
+  constructor(client: AxiosInstance) {
+    this.client = client;
+  }
+
+  /**
+   * Enumerates available custom templates for in the theme files in a specific Theme version
```

Comment:
Consistent capitalization and an extra preposition

```suggestion
   * Enumerates available custom templates for theme files in a specific theme version
```